Tasks/DownloadPackageV0/download.ts

```diff
@@ -30,8 +30,11 @@ async function main(): Promise<void> {
 	var credentialHandler = vsts.getBearerHandler(accessToken);
 	var vssConnection = new vsts.WebApi(collectionUrl, credentialHandler);
 	var coreApi = vssConnection.getCoreApi();
+	var retryLimit = parseInt(tl.getVariable("VSTS_HTTP_RETRY")) ? parseInt(tl.getVariable("VSTS_HTTP_RETRY")) : 4;
```

Comment:
Kalyan found a bug with this line of code. It wasn't handling zero well. Refer to recent check-ins on Jenkins task.